src/SshQtTestUtils/ParsePortNumberFromSocatOutputTest.cpp

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2022 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include <gtest/gtest.h>
+
+#include <array>
+#include <string_view>
+
+#include "SshQtTestUtils/ParsePortNumberFromSocatOutput.h"
+#include "TestUtils/TestUtils.h"
+
+namespace orbit_ssh_qt_test_utils {
+
+constexpr std::string_view kReferenceSocatOutput =
+    "2022/12/12 12:12:42 socat[394] N listening on AF=2 0.0.0.0:58747\n";
+constexpr std::array kInvalidSocatOutputs{
```

Comment:
Since this one is only used in the last test case, what about moving it inside the test case. No strong opinion though.

Reply:
Sounds good. Done.